frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -1,8 +1,19 @@
 <template>
   <v-container fluid class="pa-0">
     <v-form ref="form" v-model="isValidForm">
-      <div>The maximum file size is 4MB for each document. Accepted file types are jpg, jpeg, heic, png, pdf, docx, doc, xls, and xlsx.</div>
-      <AppButton v-if="!loading && !readonly" :disabled="disabled" id="add-new-file" :primary="false" size="large" width="100px" class="addFileButton" @click="addFile">Add File</AppButton>
+      <v-row v-if="header">
+        <v-col cols="3" class="pb-0">
+          <AppLabel v-if="documentType">{{ documentType }}</AppLabel>
+        </v-col>
+        <v-col cols="8" class="d-flex flex-column align-end ml-8">
+          <AppButton v-if="!loading && !readonly" :disabled="disabled" id="add-new-file" :primary="false" size="large" width="100px" class="addFileButton" @click="addFile">Add File</AppButton>
+        </v-col>
+      </v-row>
+      <template v-else>
+        <div v-if="showInfoMessage">The maximum file size is 4MB for each document. Accepted file types are jpg, jpeg, heic, png, pdf, docx, doc, xls, and xlsx.</div>
+        <AppButton v-if="!loading && !readonly" :disabled="disabled" id="add-new-file" :primary="false" size="large" width="100px" class="addFileButton" @click="addFile">Add File</AppButton>
```

Comment:
I'd like the AppButton to not be repeated but I think it needs to meet the layout requirements for both.

Reply:
Was able to use dynamic class and single button to meet both UI requirements.